src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/DocumentBasedFixAllProvider.cs

```diff
@@ -67,13 +67,14 @@ protected virtual string GetFixAllTitle(FixAllContext fixAllContext)
         public sealed override IEnumerable<FixAllScope> GetSupportedFixAllScopes()
             => _supportedFixAllScopes;
 
-        public sealed override Task<CodeAction?> GetFixAsync(FixAllContext fixAllContext)
+        public sealed override Task<CodeAction?> GetFixAsync(FixAllContext fixAllContext, IProgress<CodeAnalysisProgress> progress)
             => DefaultFixAllProviderHelpers.GetFixAsync(
-                fixAllContext.GetDefaultFixAllTitle(), fixAllContext, FixAllContextsHelperAsync);
+                fixAllContext.GetDefaultFixAllTitle(), fixAllContext, progress, FixAllContextsHelperAsync);
 
-        private Task<Solution?> FixAllContextsHelperAsync(FixAllContext originalFixAllContext, ImmutableArray<FixAllContext> fixAllContexts)
+        private Task<Solution?> FixAllContextsHelperAsync(
+            FixAllContext originalFixAllContext, ImmutableArray<FixAllContext> fixAllContexts, IProgress<CodeAnalysisProgress> progress)
             => DocumentBasedFixAllProviderHelpers.FixAllContextsAsync(originalFixAllContext, fixAllContexts,
-                    originalFixAllContext.Progress,
+                    progress,
```

Comment:
certrainly a case that got simpler.  In the past, which progrss object were you supposed to use?  The one from the original fix all context, or hte ones from the second fix-all-contexts arg?